cmd/hatchet-migrate/migrate/migrations/20251112202112_v1_0_54.sql

```diff
@@ -0,0 +1,203 @@
+-- +goose Up
+-- +goose NO TRANSACTION
+-- +goose StatementBegin
+BEGIN;
+
+CREATE TABLE v1_event_lookup_table_olap_new (
+    tenant_id UUID NOT NULL,
+    external_id UUID NOT NULL,
+    event_id BIGINT NOT NULL,
+    event_seen_at TIMESTAMPTZ NOT NULL,
+
+    PRIMARY KEY (external_id, event_seen_at)
+) PARTITION BY RANGE(event_seen_at);
+
+WITH existing_partitions AS (
+    SELECT
+        child.relname AS partition_name,
+        regexp_replace(
+            pg_get_expr(child.relpartbound, child.oid),
+            '.*FROM \(''([^'']+)''\) TO \(''([^'']+)''\).*',
+            '\1'
+        )::timestamp AS range_start
+    FROM pg_inherits
+    JOIN pg_class parent ON pg_inherits.inhparent = parent.oid
+    JOIN pg_class child ON pg_inherits.inhrelid = child.oid
+    WHERE parent.relname = 'v1_event_lookup_table_olap'
+    ORDER BY range_start
+)
+
+SELECT create_v1_weekly_range_partition('v1_event_lookup_table_olap_new'::text, range_start::DATE)
+FROM existing_partitions;
```

Comment:
thought this was a good trick - using the existing partitions to create the new ones, so we never miss any